go-controller/pkg/node/openflow_manager.go

```diff
@@ -245,7 +245,38 @@ func (c *openflowManager) updateBridgeFlowCache(hostIPs []net.IP, hostSubnets []
 	}
 	dftFlows = append(dftFlows, dftCommonFlows...)
 
-	c.updateFlowCacheEntry("NORMAL", []string{fmt.Sprintf("table=0,priority=0,actions=%s\n", util.NormalAction)})
+	if config.OvnKubeNode.Mode == types.NodeModeDPU {
+		// Retrieve the local_ip for the tunnel endpoint which will be used to create OpenFlow actions
```

Comment:
I think you should not use the encap IP here. This implies that the encap IP is the same as the br-ovn IP. Which may not always be the case. I think you should just get the IP of the br-ovn interface.